diff --git a/node_modules/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/configurators/DevLauncherExpoActivityConfigurator.kt b/node_modules/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/configurators/DevLauncherExpoActivityConfigurator.kt
index c6b9681..ec87c4b 100644
--- a/node_modules/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/configurators/DevLauncherExpoActivityConfigurator.kt
+++ b/node_modules/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/configurators/DevLauncherExpoActivityConfigurator.kt
@@ -146,8 +146,9 @@ class DevLauncherExpoActivityConfigurator(
     activity
       .window
       .addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
-    activity
-      .window.statusBarColor = color
+    if (Build.VERSION.SDK_INT < 35) {
+      activity.window.statusBarColor = color
+    }
   }
 
   fun applyNavigationBarConfiguration(activity: ReactActivity) {
@@ -158,7 +159,9 @@ class DevLauncherExpoActivityConfigurator(
     if (isValidColor(navBarColor)) {
       try {
         activity.window.clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_NAVIGATION)
-        activity.window.navigationBarColor = Color.parseColor(navBarColor)
+        if (Build.VERSION.SDK_INT < 35) {
+          activity.window.navigationBarColor = Color.parseColor(navBarColor)
+        }
       } catch (e: Throwable) {
         Log.e(TAG, "Failed to configure androidNavigationBar.backgroundColor", e)
       }
